apps/src/code-studio/pd/workshop_dashboard/workshop_index.jsx

```diff
@@ -91,6 +95,13 @@ const WorkshopIndex = React.createClass({
             >
               User Management
             </Button>}
+          {this.permission.isWorkshopAdmin &&
+            <Button
+              href={this.context.router.createHref("../../regional_partners")}
+              onClick={this.handleRegionalPartnersClick}
+            >
+              Regional Partners
```

Comment:
nit: ideally button text should have a verb. Suggestion "Manage Regional Partners"

More generally do we want to clutter this page with so many buttons? Maybe we should have a single admin page with multiple actions, and one button for that?

Reply:
@tanyaparker Any thoughts on where links to PD admin features should live such as assigning courses to facilitators and editing regional partners?

Reply:
Here ya go! https://docs.google.com/document/d/1iaMcfalaH01Mw45Aze4iFqXh7ujXgBwhb0ybLrIunlg/edit

Reply:
I've removed the Regional Partners button on the Workshop Dashboard and will implement the new Workshop Admin tools page in a separate PR.

Reply:
👍